Sources/SKCore/BuildSystem.swift

```diff
@@ -13,6 +13,9 @@
 import BuildServerProtocol
 import LanguageServerProtocol
 import TSCBasic
+import struct Foundation.URL
+
+public typealias URL = Foundation.URL
```

Comment:
This is no longer necessary.  We can just `import struct Foundation.URL` in `CompilationDatabaseBuildSystem.swift`.  The rest of the module should be clean.